modules/ding_debt_easy/ding_debt_easy.install

```diff
@@ -97,3 +104,11 @@ function ding_debt_easy_uninstall() {
   variable_del('ding_debt_easy_privacy');
   variable_del('ding_debt_easy_terms');
 }
+
+/**
+ * Add auth field to database.
+ */
+function ding_debt_easy_update_7100() {
+  $schema = ding_debt_easy_schema();
+  db_add_field('ding_debt_easy', 'auth', $schema['ding_debt_easy']['fields']['auth']);
+}
```

Comment:
If we want to avoid updating the database schema for to allow easier updated you could consider generating a signature for the payment and passing this with the payment as authentication. The validation step would then be to validate the signature.

Reply:
That is not a bad idea.... hmmm

Reply:
But the idea that it is random and not someting the end users ever would be able to generate was there to ensure that it came from Nets. And that it was not an user tha made the callback a thereby make the payment into FBS.